### PR TITLE
Do not check document links during push builds

### PR DESCRIPTION
**Pull request by rsimha-amp**
Skip checking document links during push builds, since there is no available commit range or list of files on which to work. This is fine since the document link check is already run as a presubmit for PRs.

### CODE REVIEW

File: examples/amp-by-example.html [hunk 1]
```diff
@@ -0,0 +1,1287 @@
+<!doctype html>
```
Comment (choumx) [inline examples/amp-by-example.html]: Remove this file?

Comment (rsimha-amp) [inline examples/amp-by-example.html]: Crap, that snuck in from a different branch.

Comment (rsimha-amp) [inline examples/amp-by-example.html]: Removed.

**Comment by rsimha-amp** [comment]
@choumx, could you merge this, please?

**Pull request merged**